hbase_sql/src/main/java/com/splicemachine/derby/impl/SpliceSpark.java

```diff
@@ -107,6 +125,50 @@ public static synchronized boolean isRunningOnSpark() {
         return !RegionServerLifecycleObserver.isHbaseJVM;
     }
 
+    private static UserGroupInformation applyCreds(Credentials credentials) throws IOException {
+        LOG.info("credentials:" + credentials);
+
+        logCredInformation(credentials);
+
+        UserGroupInformation ugi = UserGroupInformation.getCurrentUser();
+        ugi.addCredentials(credentials);
+        // specify that this is a proxy user
+        ugi.setAuthenticationMethod(UserGroupInformation.AuthenticationMethod.PROXY);
+
+        LOG.info("Applied credentials");
+
+        return ugi;
+    }
+
+    public static void logCredInformation(Credentials credentials2) {
+        LOG.info("credentials:" + credentials2);
+        for (int i =0 ; i < credentials2.getAllSecretKeys().size(); i++) {
+            LOG.info("getAllSecretKeys:" + i + ":" + credentials2.getAllSecretKeys().get(i));
```

Comment:
Should we perhaps log this only for debug log? I'm not sure what details secretkeys do share though.

Reply:
You are certainly right, done